aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -123,10 +123,13 @@ async def _get_derive_suite(
         self,
         *,
         wallet: BaseWallet,
-        issuer_id: str,
+        issuer_id: str = None,
     ):
         """Get signature suite for deriving credentials."""
-        did_info = await self._did_info_for_did(issuer_id)
+        if issuer_id:
+            did_info = await self._did_info_for_did(issuer_id)
+        else:
+            did_info = None
```

Comment:
verkey is not needed for derivation. See https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/tests/test_bbs_mattr_interop.py#L63-L65 for example